guide/14-deep-learning/labeling-text-using-doccano.ipynb

```diff
@@ -0,0 +1,136 @@
+{
```

Comment:
<p>Remove 12. and 13. (step numbers) in the last line, as they are not really steps. Rephrase "You can find a working sample of training workflow&nbsp;<a href="https://developers.arcgis.com/python/sample-notebooks/information-extraction-from-madison-city-crime-incident-reports-using-deep-learning/#Publishing-the-results-as-feature-layer" target="_blank">here</a>" to "You can find a sample notebook <a href="https://developers.arcgis.com/python/sample-notebooks/information-extraction-from-madison-city-crime-incident-reports-using-deep-learning/#Publishing-the-results-as-feature-layer" target="_blank">here</a></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/644/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
Made the requested changes.